src/components/UserName/UserName.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+
+const UserName = () => {
+    const [userName, setName] = useState("");
+    const handleKeydown = (event) => {
+        if (event.key === 'Enter') {
+            setName(event.target.value);
+        }
+    }
+    return (
+        <div>
```

Comment:
If we're not gonna add any classNames or property I think its better to use `fragment`
```suggestion
        <>
```